spec/fixtures/_posts/2014-03-04-march-the-fourth.md

```diff
@@ -3,7 +3,7 @@ tags:
  - '"/><VADER>'
 image:
   path: "/object-image.png"
-category: updates
+categories: updates jekyll
```

Comment:
Does this need to be an `[Array]`?

Reply:
Both notations works, default post uses this syntax:

https://github.com/jekyll/jekyll/blob/master/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb#L5